Add Command Line Interface Pages project support (#9845)
tldr-pages/tldr: pages/common/md-to-clip.md

# md-to-clip

> Converter from TlDr pages to [C]ommand [L]ine [I]nterface [P]ages.
> More information: <https://github.com/command-line-interface-pages/prototypes/tree/main/md-to-clip>.

> - Convert specific local pages and output outcome pages in the same directories:

Reviewer: ```suggestion
- Convert tldr-pages files and save into the same directories:
```
